src/main/java/com/bitalino/comm/BITalinoSocket.java

```diff
@@ -72,7 +74,12 @@ public BITalinoSocket(final DataInputStream is, final OutputStream os) {
             buffer[0] = bTemp[0];
             f = BITalinoFrameDecoder.decode(buffer, analogChannels, totalBytes);
           }
+        } else if (f.getSequence() != (prevSeq+1)%16){
+          System.out.println("Sequence out of order.")
```

Comment:
@pires Throwing an exception is maybe a bit harsh for this matter. Imagine reading chunks of 1000 samples, one of them is out of order and you end up losing 999 samples. Would it be better to just ignore this matter?